src/common.rs

```diff
@@ -100,16 +100,18 @@ pub fn color_from_string(s: &str) -> Result<rpi_led_matrix::LedColor, Box<dyn Er
 }
 
 pub fn is_connected() -> bool {
-    let response = ureq::get("http://clients3.google.com/generate_204").call();
-    info!("Checking connection, status is {:?}", response.status());
-    response.status() == 204
+    true
```

Comment:
Stonks. It’s able to connect to AWS to get scores, I assume?

Reply:
Yup, works fine

Reply:
Can you try running this just before or after getting scores?

Reply:
Wack, still returning 400s on startup. Maybe the scoreboard service is running too early, before other networking things are fully up.

Reply:
I get 204s if I start the scoreboard app manually, so I don't think its the endpoint.